### PR TITLE
BYOND.BYOND version 514.1578

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45697)

### CODE REVIEW

**Comment by FireFlashie** [comment]
Now, let's try another domain. Without forced HTTPS enabled.

**Comment by FireFlashie** [comment]
There seems to be some problem with Cloudflare and the winget check. I'll try another solution and then roll back the protocol changes afterwards.

File: manifests/b/BYOND/BYOND/514.1578/BYOND.BYOND.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BYOND.BYOND
+PackageVersion: 514.1578
+PackageLocale: en-US
+Publisher: BYOND Software
+PublisherUrl: https://www.byond.com
+PackageName: BYOND
+License: Copyright (c) BYOND Software All rights reserved.
+LicenseUrl: https://www.byond.com/TOS
```
Comment (Trenly) [inline manifests/b/BYOND/BYOND/514.1578/BYOND.BYOND.locale.en-US.yaml:11]:
```suggestion
License: Proprietary
Copyright: Copyright (c) BYOND Software All rights reserved.
CopyrightUrl: https://www.byond.com/TOS

```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by FireFlashie** [comment]
All requested changes have been made. Also, I had to replace the main URL, otherwise the Cloudflare configuration interferes with the pipeline checks, although the URL is just fine and available to users.

File: manifests/b/BYOND/BYOND/514.1578/BYOND.BYOND.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BYOND.BYOND
+PackageVersion: 514.1578
+PackageLocale: en-US
+Publisher: BYOND Software
```
Comment (OfficialEsco) [inline manifests/b/BYOND/BYOND/514.1578/BYOND.BYOND.locale.en-US.yaml:7]:
```suggestion
Publisher: BYOND
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Name        : BYOND
Publisher   : BYOND
Version     : 514.1578
ProductCode : BYOND
Uninstall   : winget uninstall --id "BYOND"
```

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**